README.md

```diff
@@ -4,76 +4,68 @@ IOpipe Analytics & Distributed Tracing Agent
-([register for access](https://www.iopipe.com)), and it will
-automatically monitor and collect metrics from your application
-running on AWS Lambda.
+Install by requiring this module, passing it an object with your project token ([register for access](https://www.iopipe.com)), and it will automatically monitor and collect metrics from your applications running on AWS Lambda.
 
-If you are using the Serverless Framework to deploy your lambdas, check out our
-[serverless plugin](https://github.com/iopipe/serverless-plugin-iopipe).
+If you are using the Serverless Framework to deploy your lambdas, check out our [serverless plugin](https://github.com/iopipe/serverless-plugin-iopipe).
 
 Example:
 
-```javascript
-var iopipe = require("iopipe")({ token: "YOUR_TOKEN"})
+```js
+var iopipe = require('iopipe')({ token: 'PROJECT_TOKEN' });
 
-exports.handle = iopipe(
-  function (event, context) {
-    context.succeed("This is my serverless function!")
-  }
-)
+exports.handler = iopipe((event, context) => {
+  context.succeed('This is my serverless function!');
+});
 ```
 
-# Environment-based config
+# Configuration
+
+#### `token` (string: required)
+
+If not supplied, the environment variable `$IOPIPE_TOKEN` will be used if present. [Find your project token](https://dashboard.iopipe.com/install)
+
+#### `timeoutWindow` (int: optional = 150)
 
-This library will look for an environment variable, `IOPIPE_TOKEN` and will use
-this if one is not explicitly passed to the configuration object. If a project
-token is passed to the configuration object, the library will prefer that
-token over the environment variable.
+By default, IOpipe will capture timeouts by exiting your function 150ms early from the AWS configured timeout, to allow time for reporting. You can disable this feature by setting `timeoutWindow` to `0` in your configuration. If not supplied, the environment variable `$IOPIPE_TIMEOUT_WINDOW` will be used if present.
 
 ```javascript
-exports.handle = require("iopipe")()(
-  function (event, context, callback) {
-    // Do things here. Nothing will be reported.
-  }
-)
+var iopipe = require('iopipe')({ token: 'PROJECT_TOKEN', timeoutWindow: 0})
 ```
 
-# Capturing timeouts
+#### `debug` (bool: optional = false)
 
-By default, IOpipe will capture timeouts by exiting your function 150ms early
-from the AWS configured timeout, to allow time for reporting. You can disable this
-feature by setting `timeoutWindow` to `0` in your configuration:
+Debug mode will log all data sent to IOpipe servers to STDOUT. This is also a good way to evaluate the sort of data that IOpipe is receiving from your application. If not supplied, the environment variable `$IOPIPE_DEBUG` will be used if present.
 
-```javascript
-var iopipe = require("iopipe")({ token: "YOUR_TOKEN", timeoutWindow: 0})
+```js
+exports.handler = require('iopipe')({
+  token: 'PROJECT_TOKEN',
+  debug: true
+})((event, context, callback) => {
```

Comment:
I think I prefer the "declare/config iopipe, then use iopipe() around handler" flow for this and the plugins sample